src/runtime/Loader.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { Component } from "react";
+
+export type LoaderProps = { errors: React.ReactNode[] };
```

Comment:
Let's rename this to RenderProps and make it an interface.

Also please always put brackets on separate lines. It helps with being able to flow through the code with your eyes, and understand the structure. You can parse faster vertically than vertically + horizontally if you know what I mean :P